Cargo.toml

```
@@ -118,6 +121,7 @@ smallvec = "1.6.1"
 libp2p-deflate = { version = "0.37.0", path = "transports/deflate", optional = true }
 libp2p-dns = { version = "0.37.0", path = "transports/dns", optional = true }
 libp2p-mdns = { version = "0.41.0", path = "protocols/mdns", optional = true }
+libp2p-quic = { version = "0.7.0-alpha", path = "transports/quic", default-features = false, optional = true }
```

That is a no-op now.

```suggestion
libp2p-quic = { version = "0.7.0-alpha", path = "transports/quic", optional = true }
```